app/src/main/res/layout/profile_chooser_profile_view.xml

```diff
@@ -96,7 +96,7 @@
       android:layout_width="match_parent"
       android:layout_height="1dp"
       android:layout_marginTop="@dimen/profile_chooser_profile_view_view_margin_top"
-      android:background="@color/oppia_profile_chooser_divider"
+      android:background="@color/component_color_profile_chooser_activity_divider_color"
```

Comment:
Can we add `component_color_profile_chooser_activity_divider_color` as `component_color_shared_activity_divider_color` What you think @rt4914  ?

Because the divider is used by most of the other fragments and activities also.